aten/src/ATen/FunctionalTensorWrapper.cpp

```diff
@@ -260,6 +260,25 @@ const char* FunctionalTensorWrapper::tensorimpl_type_name() const {
     return "FunctionalTensorWrapper";
 }
 
+at::IntArrayRef FunctionalTensorWrapper::sizes_custom() const {
```

Comment:
is there a reason you don't want to call `value_.unsafeGetTensorImpl()->sizes_custom()`, that could save us a check?

Reply:
Aha I was gonna add a comment - we could do that and save ourselves the extra check, but we'd need to make all the `*_custom` functions public